Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/AzureHttpObservable.java

```diff
@@ -57,7 +58,7 @@ public AzureHttpObservable(@NotNull String apiVersion) {
         this("common", apiVersion);
     }
 
-    public AzureHttpObservable(@NotNull String tenantId, @NotNull String apiVersion) {
+    public AzureHttpObservable(@NotNull String tenantId, @Nullable String apiVersion) {
```

Comment:
Why to change this? Could you try to initialize it in `ServerlessSparkHttpObservable`?